components/playlist/mojom/playlist.mojom

```diff
@@ -39,24 +39,33 @@ struct PlaylistItem {
   int8 last_played_position;
 };
 
-// Used by the WebUI page to bootstrap bidirectional communication.
+// Used by the web UI to bootstrap bidirectional communication.
+// the WebUI page's |BrowserProxy| singleton calls this method
+// when the page is first initialized.
+[EnableIfNot=is_android]
```

Comment:
I don't think this is really necessary, but if we want to keep them separate it would be better to put the webui specific stuff in a different mojom file

Reply:
Yep, done.